### PR TITLE
Add included field support for transactions api

### PR DESCRIPTION
**Pull request by arturdm**

### CODE REVIEW

File: sdk/src/main/java/com/meniga/sdk/models/transactions/MenigaTransaction.java [hunk 1]
```diff
@@ -397,6 +404,17 @@ public Long getAccountId() {
...
 
+	/**
+	 * @return The account this transaction belongs to.
+	 */
+	public MenigaAccount getAccount() {
+		return account;
+	}
+
+	public void setAccount(MenigaAccount account) {
+		this.account = account;
```
Comment (agustk) [inline sdk/src/main/java/com/meniga/sdk/models/transactions/MenigaTransaction.java]: The convetion in the ask is to only have setters for fields that can be updated in the back end - such as category id etc.

File: sdk/src/main/java/com/meniga/sdk/webservices/MenigaAPI.java [hunk 2]
```diff
@@ -114,7 +114,7 @@
 	@DELETE(APIConst.URL_TRANSACTIONS)
 	Call<Void> deleteTransactions(@QueryMap Map<String, String> queryMap);
 
-	@GET(APIConst.URL_TRANSACTIONS)
+	@GET(APIConst.URL_TRANSACTIONS + "?include=Account,Merchant")
```
Comment (agustk) [inline sdk/src/main/java/com/meniga/sdk/webservices/MenigaAPI.java]: Does this also support categories?

File: sdk/src/main/java/com/meniga/sdk/models/transactions/MenigaTransaction.java (hunk 1 above)
Comment (arturdm) [reply sdk/src/main/java/com/meniga/sdk/models/transactions/MenigaTransaction.java]: You’re right about those setters - I’ll try to refactor it one way or another so they’re not exposed.

File: sdk/src/main/java/com/meniga/sdk/webservices/MenigaAPI.java (hunk 2 above)
Comment (arturdm) [reply sdk/src/main/java/com/meniga/sdk/webservices/MenigaAPI.java]: They're not supported at this point.

**Pull request merged**